sdk/include/reactos/shlwapi_undoc.h

```diff
@@ -39,6 +39,29 @@ PVOID WINAPI SHInterlockedCompareExchange(PVOID *dest, PVOID xchg, PVOID compare
 LONG WINAPI SHGlobalCounterGetValue(HANDLE hGlobalCounter);
 LONG WINAPI SHGlobalCounterIncrement(HANDLE hGlobalCounter);
 
+#if FALSE && DLL_EXPORT_VERSION >= _WIN32_WINNT_VISTA
+#define SHELL_GCOUNTER_DEFINE_GUID(name, a, b, c, d, e, f, g, h, i, j, k) enum { SHELLUNUSEDCOUNTERGUID_##name }
+#define SHELL_GCOUNTER_DEFINE_HANDLE(name) enum { SHELLUNUSEDCOUNTERHANDLE_##name }
+#define SHELL_GCOUNTER_PARAMETERS(handle, id) id
+#define SHELL_GlobalCounterCreate(refguid, handle) ( (refguid), (handle), (void)0 )
+#define SHELL_GlobalCounterIsInitialized(handle) ( (handle), TRUE )
+#define SHELL_GlobalCounterGet(id) SHGlobalCounterGetValue_Vista(id)
+#define SHELL_GlobalCounterIncrement(id) SHGlobalCounterIncrement_Vista(id)
+#else
+#define SHELL_GCOUNTER_DEFINE_GUID(name, a, b, c, d, e, f, g, h, i, j, k) const GUID name = { a, b, c, { d, e, f, g, h, i, j, k } }
```

Comment:
Just to understand, we cannot use the `DEFINE_GUID` macro here?

Reply:
DEFINE_GUID depends on the "state" of the initguid.h header. These global counter GUIDs are only used in a single place and defining them like this should be OK.